include/deal.II/matrix_free/cuda_hanging_nodes_internal.h

```diff
@@ -282,7 +283,7 @@ namespace CUDAWrappers
 
       std::vector<types::global_dof_index> neighbor_dofs(dofs_per_face);
 
-      std::vector<types::global_dof_index> lex_face_mapping =
+      const auto lex_face_mapping =
         FETools::lexicographic_to_hierarchic_numbering<dim - 1>(
```

Comment:
The change is not wrong, but out of curiosity: what was the reason to change this? Isn't the type always `std::vector<unsigned int>`? (I.e., it was clearly wrong before.)

Reply:
Yes, it is. At that point, I was just happy to let the compiler choose the right type and it is shorter.
Call it laziness. I am not sure if we have a coding standard in this regard.

Reply:
No. I typically err on the side of being explicit unless the type is obvious (e.g., for cells). But I'm happy to leave as is :-)